tests/KlaviyoAPITest.php

```diff
@@ -0,0 +1,271 @@
+<?php
+
+
+use Klaviyo\KlaviyoAPI as Klaviyo;
+use PHPUnit\Framework\TestCase;
+
+class KlaviyoAPITest extends TestCase
+{
+    public $testKlaviyoClass;
+    public $testAPI;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+    }
+
+    public function testPublicKey()
+    {
+        $this->testKlaviyoClass = new class('Test0A', 'pk_testprivatekey') extends Klaviyo{
+
+            public function returnPublicKey( ){
+                return $this->public_key;
+            }
+        };
+
+        $this->assertEquals( 'Test0A', $this->testKlaviyoClass->returnPublicKey() );
+
+    }
+
+    public function testPrivateKey()
+    {
+        $this->testKlaviyoClass = new class('Test0A', 'pk_testprivatekey') extends Klaviyo{
+
+            public function returnPrivateKey( ){
+                return $this->private_key;
+            }
+        };
+
+        $this->assertEquals( 'pk_testprivatekey', $this->testKlaviyoClass->returnPrivateKey() );
+
+    }
+
+    public function testClient()
+    {
+        $this->testKlaviyoClass = new class('Test0A', 'pk_testprivatekey') extends Klaviyo{
+
+            public function returnClient(){
+                return $this->client;
+            }
+        };
+
+        $this->assertInstanceOf(\GuzzleHttp\Client::class, $this->testKlaviyoClass->returnClient() );
+
+    }
+
+    public function testPublicAuth()
+    {
+        $this->testKlaviyoClass = new class('Test0A', 'pk_testprivatekey') extends Klaviyo{
+
+            public function returnPublicAuth( $params ){
+                return $this->publicAuth( $params );
+            }
+        };
+
+        $methodInput = array(
+            'headers' => array(
+                'api-key' => ''
+            ),
+            'query' => array()
+        );
+
+        $expected = array(
+            'query' => array(
+                'data' => 'eyJ0b2tlbiI6IlRlc3QwQSJ9'
+            )
+        );
+
+        $this->assertEquals( $expected, $this->testKlaviyoClass->returnPublicAuth( $methodInput ));
+    }
+
+    public function testV1Auth()
+    {
+        $this->testKlaviyoClass = new class('Test0A', 'pk_testprivatekey') extends Klaviyo {
+
+            public function returnV1Auth( $params )
+            {
+                return $this->v1Auth( $params );
+            }
+        };
+
+        $expected = array(
+            'query' => array(
+                'api_key' => 'pk_testprivatekey'
```

Comment:
are you able to store 'pk_testprivatekey' somewhere and then reference it instead of having it hardcoded separate times?